client/src/controllers/ActionController.ts

```diff
@@ -103,8 +103,13 @@ export class ActionController extends Controller<
    * Select all the text in an input or textarea element.
    */
   select() {
-    if (this.element instanceof HTMLButtonElement) return;
-    this.element?.select();
+    if (
+      this.element &&
+      (this.element instanceof HTMLInputElement || this.element instanceof HTMLTextAreaElement) &&
+      typeof this.element.select === 'function'
+    ) {
+      this.element.select();
+    }
```

Comment:
```suggestion
    const element = this.element;

    if (
      element instanceof HTMLInputElement ||
      element instanceof HTMLTextAreaElement
    ) {
      element.select();
    }
  }
```

A few notes

1. If we are using `element` multiple times, let's pull it out as a variable, this is a bit simpler to follow in the code this way.
2. checking instance of should be enough to get to the goal here, so let's avoid checking the type of the function and whether this.element exists.